scylla-server/README.md

```diff
@@ -16,24 +10,34 @@ docker compose up odyssey-timescale
 
 ```
 # in this directory
-SOURCE_DATABASE_URL=postgresql://postgres:password@127.0.0.1:5432/postgres cargo run
+cargo run
 ```
 
 ### Test this app
 
-#### Seed some data
+#### Get started with DB
+
+Download the diesel CLI:
+```
+# Linux/MacOS
+curl --proto '=https' --tlsv1.2 -LsSf https://github.com/diesel-rs/diesel/releases/latest/download/diesel_cli-installer.sh | sh
+
+# Windows
+powershell -c "irm https://github.com/diesel-rs/diesel/releases/latest/download/diesel_cli-installer.ps1 | iex"
```

Comment:
Would the install guide on this page work? These specific download commands feels like they're gonna be prone to breaking without much guidance on other options, wanna make the setup guide as easy to follow as possible:
https://diesel.rs/guides/getting-started